server/game/cards/01-DTR/ThePharmacy.js

```diff
@@ -0,0 +1,25 @@
+const DeedCard = require('../../deedcard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class ThePharmacy extends DeedCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'The Pharmacy',
+            playType: ['noon'],
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Select dude to unboot',
+                cardCondition: { location: 'play area', booted: true },
+                cardType: ['dude']
+            },
+            message: context => {this.game.addMessage('{0} uses {1} to unboot {2}', context.player, this, context.target);},
```

Comment:
```suggestion
            message: context => 
                this.game.addMessage('{0} uses {1} to unboot {2}', context.player, this, context.target),
```
This is how it should look like to remove problem with formatting. You can check your change is correctly formatted by running `npm run lint` in the townsquare repo.

Reply:
If you do not want to do it yourself, you can just click on `Commit suggestion` button above.